msteams-platform/concepts/device-capabilities/browser-device-permissions.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Device permissions for the browser
+keywords: teams apps capabilities permissions
+description: Securely bring back device permissions support for apps in our web client
+localization_priority: Normal
+ms.topic: how-to
+---
+
+# Device permissions for the browser
+
+Users can grant consent to each app and consent to device permissions. Currently, when permissions are granted for one app, it's also granted for all other apps. An update made in Chromium browser deprecated HTML5 permissions from cross-origin iframes. Chromium provides a way to allow iframes to request these permissions:
+
+* Camera
+* Microphone
+* Location access
+
+The request or consent dialog box is for the ancestor in the browser. For example, if an app grants microphone access, other apps automatically grants device permissions for microphone. The user will never see any prompt from this app as the permission is already granted.
```

Comment:
What request or consent dialog box are we talking about? This paragraph is misleading. We should remove or rephrase this. Also the flow diagram needs to be enhanced to include the consent flow. It only has the deny flow right now.

Reply:
It looks like maybe you're trying to state a problem from the previous Chromium experience. This is not necessary and is confusing to the user. You should focus on the present experience instead.